core/src/com/unciv/models/ruleset/Difficulty.kt

```diff
@@ -31,36 +34,46 @@ class Difficulty: INamed {
 
-        return lines.joinToString("\n") { it.tr() }
+    override var civilopediaText = listOf<FormattedLine>()
+
+//    fun getDescription(): String {
+//        return getCivilopediaTextLines(RulesetCache.getBaseRuleset()).joinToString("\n") { it.text.tr() }
+//    }
+
+    override fun makeLink() = "Difficulty/$name"
+    override fun replacesCivilopediaDescription() = true
+    override fun hasCivilopediaTextLines() = true
+
+    private fun Float.toPercent() = (this * 100).toInt()
+    override fun getCivilopediaTextLines(ruleset: Ruleset): List<FormattedLine> {
+        val lines = ArrayList<FormattedLine>()
+        lines += FormattedLine("Player settings", header = 3)
+        lines += FormattedLine("{Base Happiness}: $baseHappiness ${Fonts.happiness}", indent = 1)
+        lines += FormattedLine("{Extra happiness per luxury}: ${extraHappinessPerLuxury.toInt()} ${Fonts.happiness}", indent = 1)
+        lines += FormattedLine("{Research cost modifier}: ${researchCostModifier.toPercent()}% ${Fonts.science}", indent = 1)
+        lines += FormattedLine("{Unit cost modifier}: ${unitCostModifier.toPercent()}% ${Fonts.production}", indent = 1)
+        lines += FormattedLine("{Building cost modifier}: ${buildingCostModifier.toPercent()}% ${Fonts.production}", indent = 1)
+        lines += FormattedLine("{Policy cost modifier}: ${policyCostModifier.toPercent()}% ${Fonts.culture}", indent = 1)
+        lines += FormattedLine("{Unhappiness modifier}: ${unhappinessModifier.toPercent()}%", indent = 1)
+        lines += FormattedLine("{Bonus vs. Barbarians}: ${barbarianBonus.toPercent()}% ${Fonts.strength}", indent = 1)
+//        lines += FormattedLine("{Starting units}: $startingUnits", indent = 1)
+        lines += FormattedLine()
+        lines += FormattedLine("AI settings", header = 3)
+        lines += FormattedLine("{AI city growth modifier}: ${aiCityGrowthModifier.toPercent()}% ${Fonts.food}", indent = 1)
+        lines += FormattedLine("{AI unit cost modifier}: ${aiUnitCostModifier.toPercent()}% ${Fonts.production}", indent = 1)
+        lines += FormattedLine("{AI building cost modifier}: ${aiBuildingCostModifier.toPercent()}% ${Fonts.production}", indent = 1)
+        lines += FormattedLine("{AI wonder cost modifier}: ${aiWonderCostModifier.toPercent()}% ${Fonts.production}", indent = 1)
+        lines += FormattedLine("{AI building maintenance modifier}: ${aiBuildingMaintenanceModifier.toPercent()}% ${Fonts.gold}", indent = 1)
+        lines += FormattedLine("{AI unit maintenance modifier}: ${aiUnitMaintenanceModifier.toPercent()}% ${Fonts.gold}", indent = 1)
+//        lines += FormattedLine("{AI free techs}: $aiFreeTechs", indent = 1)
+//        lines += FormattedLine("{AI major civilizations starting units}: $aiMajorCivStartingUnits", indent = 1)
+//        lines += FormattedLine("{AI city-state starting units}: $aiCityStateStartingUnits", indent = 1)
```

Comment:
Why not?

Reply:
Was commented out before, I just moved the block and did a replace on them all so they look like I re-created them but actually - not. We can go look for the why and what additional info would be OK later.

Reply:
Actually, I'm trying the unlisted properties now.